documentation/Composition methods.tex

```diff
@@ -25,7 +21,7 @@ \section{Differential equations.}
 \quad\parbox{.4\linewidth}{with $\exp h\vA$ and $\exp h\vB$ known and
 $\commutator{\vB}{\commutator{\vB}{\commutator{\vB}{\vA}}}=\nullvec$;}
 \label{general}\\
-&\quad\text{the above equation, with $\exp h\vA = \Identity+h\vA$,
+\span\parbox{.7\linewidth}{the above equation, with $\exp h\vA = \Identity+h\vA$,
```

Comment:
Please let's not go around tweaking pixels.  The original code was just fine AFAIAC.  Not sure why it had to be changed, but I don't want to have to write crud like that.

Reply:
> The original code was just fine

The original code *was* doing pixel-tweaking: this line is not aligned with the equalities in the `align`, and `&\quad` was manually spacing it by one em with respect to the alignment point (whereas `\span` makes it unaligned).

Similarly `\text` relied on the text fitting on one line, whereas `\parbox` allows linebreaking.